packages/next/client/future/image.tsx

```diff
@@ -270,6 +272,34 @@ function handleLoading(
       onLoadingCompleteRef.current({ naturalWidth, naturalHeight })
     }
     if (process.env.NODE_ENV !== 'production') {
+      if (img.getAttribute('data-nimg') === 'future-fill') {
+        if (
+          !img.getAttribute('sizes') ||
+          img.getAttribute('sizes') === '100vw'
+        ) {
+          let widthViewportRatio =
+            img.getBoundingClientRect().width / window.innerWidth
+          if (widthViewportRatio < 0.6) {
+            warnOnce(
+              `Image with src "${src}" has no sizes prop, but doesn't take up the full width of the screen. Use the sizes prop (https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) to improve page performance.`
+            )
+          }
+        }
+        if (img.parentElement) {
+          const { position } = window.getComputedStyle(img.parentElement)
+          if (position !== 'relative') {
```

Comment:
```suggestion
          if (position === 'static') {
```

The docs said "non-static CSS" so should we change this warning too?